resharper/resharper-unity/src/CSharp/Feature/Services/CallGraph/ShowCallsBulbActionBase.cs

```diff
@@ -0,0 +1,91 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using JetBrains.Application.Threading;
+using JetBrains.ProjectModel;
+using JetBrains.ReSharper.Feature.Services.Bulbs;
+using JetBrains.ReSharper.Feature.Services.CallHierarchy.FindResults;
+using JetBrains.ReSharper.Features.Inspections.CallHierarchy;
+using JetBrains.ReSharper.Psi;
+using JetBrains.TextControl;
+using JetBrains.UI.Icons;
+using JetBrains.Util.DataStructures.Collections;
+
+namespace JetBrains.ReSharper.Plugins.Unity.CSharp.Feature.Services.CallGraph
+{
+    public abstract class ShowCallsBulbActionBase : IBulbAction
+    {
+        public void Execute(ISolution solution, ITextControl textControl)
+        {
+            solution.Locks.AssertReadAccessAllowed();
+
+            var text = Text;
+
+            if (!solution.GetPsiServices().Caches.WaitForCaches(text))
```

Comment:
Check that SWA is completed at least once after start (there is property in solutionanalysisconfiguration), if swa is not completed, show tooltip